server/bot.go

```diff
@@ -28,16 +28,13 @@ func (p *Plugin) PostBotDM(userID string, message string) error {
 }
 
 // PostBotCustomDM posts a DM as the cloud bot user using custom post with action buttons.
-func (p *Plugin) PostBotCustomDM(userID string, message string, todo string, issueID string) error {
+func (p *Plugin) PostBotCustomDM(userID string, message string, todo string, issueID string) {
 	channel, appError := p.API.GetDirectChannel(userID, p.BotUserID)
-	if appError != nil {
-		return appError
-	}
-	if channel == nil {
-		return fmt.Errorf("could not get direct channel for bot and user_id=%s", userID)
+	if appError != nil || channel == nil {
+		return
```

Comment:
Sorry, I was ambiguous with my request. The error should not be ignored but surely logged to allow proper debugging.

Reply:
Done!